distribution/ddf/pom.xml

```diff
@@ -258,6 +258,11 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
+            <groupId>ddf</groupId>
```

Comment:
❓ Was this discovered when you bumped the version?

Reply:
Yes. In a multi-threaded build the ddf distribution was getting built in parallel with the docs.